### ISSUE TITLE
BingX fetch_tickers() every 'change' and 'percentage' value is non-negative

### ISSUE THREAD

**Issue by JovianMoon** [ccxt/ccxt#20417]
### Operating System

Linux Mint

### Programming Languages

Python

### CCXT Version

4.1.91

### Description

Every single 'change' and 'percentage' value provided by `fetch_tickers` method for BingX have non-negative values. The problem is clearly in the numbers provided by BingX API, as can be seen in the 'priceChange' and 'priceChangePercent' fields. 

It looks like their 'priceChange' value equals to 'high' minus 'low', which is not what the 'change' for the ccxt dictionary should be.

### Code

```
exchange = ccxt.bingx()
print(exchange.fetch_tickers())
```
```
{'BTC/USDT': {'symbol': 'BTC/USDT', 'timestamp': 1702913630828, 'datetime': '2023-12-18T15:33:50.828Z', 'high': 42422.4, 'low': 40550.71, 'bid': 41324.71, 'bidVolume': 3.40211, 'ask': 41325.55, 'askVolume': 3.44553, 'vwap': 41342.30311787659, 'open': 41662.74, 'close': 41325.33, 'last': 41325.33, 'previousClose': None, 'change': 1871.69, 'percentage': 4.49, 'average': 41494.035, 'baseVolume': 6225.07, 'quoteVolume': 257358730.87, 'info': {'symbol': 'BTC-USDT', 'openPrice': '41662.74', 'highPrice': '42422.40', 'lowPrice': '40550.71', 'lastPrice': '41325.33', 'priceChange': '1871.69', 'priceChangePercent': '4.49%', 'volume': '6225.07', 'quoteVolume': '257358730.87', 'openTime': '1702827230828', 'closeTime': '1702913630828', 'askPrice': '41325.55', 'askQty': '3.44553', 'bidPrice': '41324.71', 'bidQty': '3.40211'}}
```

**Comment by carlosmiei** [comment]
Hello @JovianMoon, you're right according to ccxt's definition `change` should be computed by doing `last - open` and not `high - low`, we will probably remove it from the unified structure. Thanks for letting us know.

**Comment by JovianMoon** [comment]
@carlosmiei BingX 'priceChangePercent' field, on the other hand, seems to equal to 'priceChange' / 'open', giving also an incorrect value. So 'percentage' field in the ccxt dictionary should also not copy this value. This is not currently the case in pr #20418

**Comment by carlosmiei** [comment]
> @carlosmiei BingX 'priceChangePercent' field, on the other hand, seems to equal to 'priceChange' / 'open', giving also an incorrect value. So 'percentage' field in the ccxt dictionary should also not copy this value. This is not currently addressed in pr #20418

@JovianMoon yeah agreed
